### DISCUSSION TITLE
Languages

### DISCUSSION THREAD

**Discussion by boroicamarius** [Animenosekai/translate#42] [Q&A] [answered]
How many languages does this api have?

**Comment by Animenosekai** [comment, marked as answer]
@boroicamarius it depends on the API used.

Using the `Translate` class, you should get as many languages as possible (you can look at [each translator individually](https://github.com/Animenosekai/translate/tree/main/translatepy/translators), they all have a `_supported_languages` attribute with the supported languages)
